util/cookies.ts

```diff
@@ -0,0 +1,33 @@
+import { serialize } from 'cookie';
+
+export function createSerializedRegisterSessionTokenCookie(token: string) {
+  // Use secure cookies in production (HTTPS only)
+  const isProduction = process.env.NODE_ENV === 'production';
+
+  // Set cookie expiration to 24 hours
+  const maxAge = 60 * 60 * 24;
+
+  return serialize('sessionToken', token, {
+    // Set max age for modern browsers
+    maxAge: maxAge,
+    // Set max age for older browsers
+    expires: new Date(Date.now() + maxAge * 1000),
+    httpOnly: true,
+    secure: isProduction,
+    path: '/',
+    sameSite: 'lax',
```

Comment:
This can be ordered like [the cookie options in the Next.js docs](https://nextjs.org/docs/canary/app/api-reference/functions/cookies#options) and documented better

You can use the version over here:

- https://github.com/upleveled/next-js-example-fall-2024-atvie/pull/10

Reply:
[5a205ab](https://github.com/upleveled/expo-example-fall-2024-atvie/pull/1/commits/5a205abbb49c3eb76982f4177b159bfbca6edb9f)

Reply:
We're missing the 2 inline comments from https://github.com/upleveled/next-js-example-fall-2024-atvie/pull/10:

```ts
export const secureCookieOptions = {
  maxAge: 60 * 60 * 24, // Cookie expires in 24 hours
  secure: process.env.NODE_ENV === 'production',
  httpOnly: true,
  sameSite: 'lax', // Prevent sending cookie with images or frames of your content originating on other websites
} as const;
```

Reply:
Added comments 
[c625e1c](https://github.com/upleveled/expo-example-fall-2024-atvie/pull/1/commits/c625e1c2c21720b15143e755ba652e1a086ad8fe)